### PR TITLE
feat(parameter-pane): create components for parameter pane

### PR DESCRIPTION
**Pull request by esterTion**
Components can be used for parameter pane. 

* TODO: 
  * Refactor the code from `antd` to use self styled components

### CODE REVIEW

**Comment by Lutra-Fs** [comment]
I am still confused about the scope of this PR and the linked issue. Should we implement the design by @peipacut and replace the current design rather than only create components? Dropping using `antd` could be a decision by @LorenzoValentine, and we could then have a PR for the whole codebase.

**Comment by LorenzoValentine** [comment]
> I am still confusing about the scope of this PR and the linked issue. Should we implement the design by @peipacut and replace the current design rather than only create components? Dropping using `antd` could be a decision by @LorenzoValentine, and we could then have a PR for the whole codebase.

Yea, you are right. The refactoring of the current design includes moving the current `setColor` method to the new design. And it should try to be implemented according to the prototype. The `antd` you can use at the moment, and in the future, we will use new branches to refactor all the existing buttons.

File: src/components/ParameterComponents/DropdownParameter.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	)
+	/*
+	return (
+	<select value={value} onChange={e => (setValue(e.target.value), props.onChange(e.target.value))}>
+		{props.values.map((val, i) => (
+			<option key={i} value={val} >{val}</option>
+		))}
+	</select>
+	);
+	*/
```
Comment (LorenzoValentine) [inline src/components/ParameterComponents/DropdownParameter.tsx:25]: If those codes will not be used in the future, please remove them.

**Comment by esterTion** [comment]
update: 

Components refactor is complete, except for `ParameterLabel`

**Review by LorenzoValentine — APPROVED** [review]

**Pull request merged**